cloudinit/config/schema.py

```diff
@@ -494,7 +516,11 @@ def get_meta_doc(meta: MetaSchema, schema: dict) -> str:
 
     # cast away type annotation
     meta_copy = dict(deepcopy(meta))
-    meta_copy["property_doc"] = _get_property_doc(schema)
+    try:
+        meta_copy["property_doc"] = _get_property_doc(schema)
+    except AttributeError:
+        LOG.warning("Unable to render property_doc due to invalid schema")
+        meta_copy["property_doc"] = ""
```

Comment:
If our schema doesn't include properties or otherwise doesn't adhere to the metaschema, this call would fail with a very obscure traceback rather than getting the test failure from the metaschema test telling us what we did wrong.

Reply:
I like the addition of a useful message. Do you think we might want to add a unittest ensuring this isn't in the logs?

Reply:
That's actually a little easier said than done. Since `get_meta_doc` runs at import time through various `__doc__ = get_meta_doc(meta, schema)` calls, we'll see this warning before the test has fully initialized and before the caplog fixture has a chance to start capturing logs.

Since this is really only a problem if we've defined a schema that fails against the metaschema, and since we test all of the module schemas against the metaschema, I'm ok leaving this one as is.